dds/DCPS/Serializer.h

```diff
@@ -429,7 +429,7 @@ class OpenDDS_Dcps_Export Serializer {
   /// skipping the n * size bytes.
   /// This is used by the RTPS protocol to allow reading messages from
   /// future versions of the spec which may have additional optional fields.
-  bool skip(ACE_CDR::UShort n, int size = 1);
+  bool skip(size_t n, int size = 1);
```

Comment:
I kept wanting to change that, but never had an excuse to.

Reply:
The sheer number of warnings must have contributed to the "out of disk space" errors on Azure Pipelines.